packages/sources/finage/src/endpoint/uk-etf.ts

```diff
@@ -1,51 +1,12 @@
-import type { Config, ExecuteWithConfig, InputParameters } from '@chainlink/ea-bootstrap'
-import { Requester, util, Validator } from '@chainlink/ea-bootstrap'
-import { NAME } from '../config'
-import overrides from '../config/symbols.json'
+import { makeEtfExecute } from './etf'
+import { inputParameters as etfInputParameters } from './etf'
 
 export const supportedEndpoints = ['uk_etf']
 export const batchablePropertyPath = [{ name: 'base' }]
 
+export const inputParameters = etfInputParameters
```

Comment:
is this used anywhere?

Reply:
Yes! This tripped me up at first - the V2 framework relies on certain properties being exported from the file, and this is one of them (else the README generation breaks).